src/ssl.c

```diff
@@ -8728,6 +8728,19 @@ int wolfSSL_CTX_set_cipher_list(WOLFSSL_CTX* ctx, const char* list)
 int wolfSSL_set_cipher_list(WOLFSSL* ssl, const char* list)
 {
     WOLFSSL_ENTER("wolfSSL_set_cipher_list");
+#ifdef SINGLE_THREADED
+    if (ssl->ctx->suites == ssl->suites) {
+        ssl->suites = NULL;
```

Comment:
Remove line - assignment below will ensure the value is set to NULL or a valid pointer.

Reply:
Done.